vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,51 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+    var state = "normal";
+    var commaPos = 0;
+    for (var i=0;i<value.length;++i) {
+        if (state === "normal") {
```

Comment:
@zpao I fixed those in all the places I found; is there a style linter I can use to check for this in the future?